packages/nextjs/app/builders/page.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import Link from "next/link";
+import type { NextPage } from "next";
+import { MagnifyingGlassPlusIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+
+const Builders: NextPage = () => {
+  const { data: checkedInEvents, isLoading: isCheckedInEventsLoading } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 120936875n,
+  });
+
+  return (
+    <>
+      <div className="flex items-center flex-col flex-grow pt-10">
+        <div className="px-5">
+          <h1 className="text-center">
+            <span className="block text-4xl font-bold">Get to know Batch 6 builders!!</span>
+          </h1>
+        </div>
+        <div className="flex items-center flex-col flex-grow">
+          {isCheckedInEventsLoading ? (
+            <div className="flex justify-center items-center mt-10">
+              <span className="loading loading-spinner loading-lg"></span>
+            </div>
+          ) : (
+            <div className="mt-8">
+              <div className="overflow-x-auto shadow-lg">
+                <table className="table table-zebra w-full">
```

Comment:
Maybe also add `bg-base-100` here ? sicne on dark mode the table kind of get merged with background

Reply:
I added the `bg-base-100` property directly in the td tags:

![image](https://github.com/BuidlGuidl/batch6.buidlguidl.com/assets/48974550/a02749b5-0cb1-40a7-bcea-f33baef84967)